src/ontology/mondo-edit.obo

```diff
@@ -288886,14 +289367,13 @@ synonym: "autosomal recessive nonsyndromic deafness type 105" NARROW [DOID:01104
 synonym: "autosomal recessive nonsyndromic deafness type 32" NARROW [DOID:0110491, MONDORULE:2]
 synonym: "CDC14A autosomal recessive nonsyndromic deafness" NARROW [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
 synonym: "deafness, autosomal recessive 105" NARROW [OMIM:616958]
-synonym: "deafness, autosomal recessive 105, formerly" EXACT [OMIM:608653]
```

Comment:
@joeflack4 it looks like `deafness, autosomal recessive 105` should be represented as `RELATED` based on https://omim.org/entry/608653. Why was `synonym: "deafness, autosomal recessive 105" NARROW [OMIM:616958]` maintained instead of adding this synonym as `RELATED`?